src/index.css

```diff
@@ -1,13 +1,13 @@
 body {
```

Comment:
Do we need this file now as well ?

Reply:
yes, font issues will be there, if we ignore this

Reply:
You can define this using emotion also.